src/main/java/teammates/common/datatransfer/FeedbackRubricQuestionDetails.java

```diff
@@ -469,15 +471,23 @@ public String getQuestionResultStatisticsHtml(List<FeedbackResponseAttributes> r
         String tableBodyFragmentTemplate = FeedbackQuestionFormTemplates.RUBRIC_RESULT_STATS_BODY_FRAGMENT;
         String tableBodyTemplate = FeedbackQuestionFormTemplates.RUBRIC_RESULT_STATS_BODY;
         DecimalFormat df = new DecimalFormat("#"); 
+        DecimalFormat dfAverage = new DecimalFormat("###.##");
         
-        for(int j = 0 ; j < numOfRubricSubQuestions ; j++) {
+        for (int j = 0; j < numOfRubricSubQuestions; j++) {
             StringBuilder tableBodyFragmentHtml = new StringBuilder();
-            for(int i = 0 ; i < numOfRubricChoices ; i++) {
+            for (int i = 0; i < numOfRubricChoices; i++) {
                 String tableBodyCell = 
                         FeedbackQuestionFormTemplates.populateTemplate(tableBodyFragmentTemplate,
-                                "${percentageFrequency}", df.format(rubricStats[j][i]*100) + "% (" + responseFrequency[j][i] +")");
+                                "${percentageFrequencyOrAverage}", 
+                                df.format(rubricStats[j][i]*100) + "% (" + responseFrequency[j][i] +")");
```

Comment:
Here too. More specifically, you can move `df.format(...) + "%"` to the same line as the `${...}` and everything else after a line break, so that the hierarchy is similar to the previous ones.